Snowflake.Data/Core/Authenticator/ExternalBrowserAuthenticator.cs

```diff
@@ -187,7 +206,7 @@ private static void StartBrowser(string url)
             // The following code is learnt from https://brockallen.com/2016/09/24/process-start-for-urls-on-net-core/
 #if NETFRAMEWORK
             // .net standard would pass here
-            Process.Start(url);
+            Process.Start(new ProcessStartInfo(url) { UseShellExecute = true });
```

Comment:
why this change?

Reply:
I encountered the error `System.ComponentModel.Win32Exception: The system cannot find the file specified` while I was testing locally. And setting the `UseShellExecute` property to true fixed the error